*: update contributing guide (#4325)
tikv/tikv: CONTRIBUTING.md

The project now enables [DCO check](https://github.com/probot/dco#how-it-works) and the commit message must contain a `Signed-off-by` line for [Developer Certificate of Origin](https://developercertificate.org/)

You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 

### Handling CI

We use an ad-hoc jenkins CI deployment to test each PR. If you are submitting a

> PR for the first time, you will need someone of us to comment`/ok-to-test` to

Reviewer: ```suggestion
PR for the first time, you will need someone of us to comment `/ok-to-test` to
```

> PR for the first time, you will need someone of us to comment `/ok-to-test` to

Reviewer 2: seem the first commit can still trigger the CI, right?

Author: No, for security reasons, only users in the whitelist can trigger the CI.

trigger the CI. After this, new changes on the branch will be able to trigger

> the test automatically. You may also manually run the test by commenting

Reviewer: ```suggestion
the test automatically. You can also manually run the test by commenting
```

> the test automatically. You can also manually run the test by commenting

Reviewer 3: ```suggestion
the test automatically. You will also be able manually run the test by commenting
```

Author: Signed-off-by: disksing <i@disksing.com>

> `/test`.

Reviewer 4: @zhouqiang-cl Is `/rebuild` and `/test` the same?

Author: Tried `/rebuild`, seems to be the same.
